crates/metadata/src/lib.rs

```diff
@@ -57,6 +57,30 @@ use serde::{
     Serialize,
 };
 
+/// Versioned ink! project metadata.
+///
+/// # Note
+///
+/// Represents the version of the serialized metadata *format*, which is distinct from the version
+/// of this crate for Rust semver compatibility.
```

Comment:
```suggestion
/// of this crate for Rust semantic versioning compatibility.
```
Pretty sure that `cargo-spellcheck` will complain otherwise.